### ISSUE TITLE
"plugin has crashed" error

### ISSUE THREAD

**Issue by FF7CloudsNobody** [Ertego/gog-galaxy-discord#7]
Getting an error trying to link discord using this plugin
![image](https://user-images.githubusercontent.com/20816163/89413247-80d59000-d6f6-11ea-818b-72defc19a017.png)
[plugin-discord-ce69028e-a7e4-462a-96d8-468259c298e2.log](https://github.com/Ertego/gog-galaxy-discord/files/5028393/plugin-discord-ce69028e-a7e4-462a-96d8-468259c298e2.log)

**Comment by lele-blue** [comment]
It seems like the method to extract the localstorage of discord returns null.
This might be discord again trying to hide it.

I'll try to look into that issue soon.

**Comment by antv199** [comment]
Sorry for bothering, but do we have any news regarding this? For me, the error is caused by an error, in regards to the Python WebSocket module not existing. I tried downloading it via pip, but nothing changed.

Looking at the latest release, it doesn't contain it. And this is a fresh download.
![Screenshot](https://user-images.githubusercontent.com/25984671/94942935-a48f3980-04df-11eb-8301-a81295fa2f92.png)


[plugin-discord-ce69028e-a7e4-462a-96d8-468259c298e2.log](https://github.com/Ertego/gog-galaxy-discord/files/5318962/plugin-discord-ce69028e-a7e4-462a-96d8-468259c298e2.log)

**Comment by lele-blue** [comment]
@antv199 I stopped developing the extension since discord stopped selling games, so i can only give limited support as don't really know how the code works anymore. It did work in this configuration once though.
If you install the websocket module via pip, did you specify the path of the extension in the gog "installed" folder? Globally installed modules aren't working inside gog

**Comment by antv199** [comment]
@Ertego Sorry for not being specific. At first, I did the usual `pip install websocket`, but now that you mentioned the installation within the installed folder, I downloaded the websocket module within the release and then I replaced its contents to the installed directory for the plugin itself.

`pip install websocket -t \discord`

And now it seems to be working just fine (without that error, at least). It only needed the websocket module, that was missing from the release, I guess.

**Comment by lele-blue** [comment]
It certainly seems odd, i'll pin this issue to point to your fix until it is fixed :)

**Comment by FF7CloudsNobody** [comment]
@antv199 I'm trying to follow what you did but I am still having issues, are you moving all the downloaded files from the pip command to this plugin's folder and replacing everything or are you only moving certain things from the download?

**Comment by antv199** [comment]
@FF7CloudsNobody all i did was download the plugin and then within the plugin's folder `C:\Users\[whatever your profile name is]\AppData\Local\GOG.com\Galaxy\plugins\installed` and then opened a cmd within that folder and did `pip install websocket -t \discord`

But here's the thing. the plugin hasn't received an update in a year, so chances are that the Discord API is outdated.

**Comment by lele-blue** [comment]
> the plugin hasn't received an update in a year, so chances are that the Discord API is outdated.

That is true, and since discord shut down the storefront I don't plan to update the code. It may still work though.
